src/accounts.rs

```diff
@@ -32,51 +38,211 @@ pub struct ErrorCounters {
     pub missing_signature_for_fee: usize,
 }
 
+//
+// A persistent account is stored in below format:
+//  account_path/--+
+//                 +-- <DIR> <== concatenated instances of
```

Comment:
This doesn't make sense to me.

Reply:
since the Account can be variable length due to presence of userdata, each account has a length field prepended to it to help traverse the data

Reply:
Yea, i kind of get the data file serialization layout, it's the directory paths I don't get. Seems like it should be `<path>/<pid>/data` where `<path>` comes from `SOLANA_ACCOUNT_PATHS` or is set to `[0,1,2,3]` by default.

I just don't know what the arrow pointed to `<DIR>` means or the `--+ +--` or a directory called `account_path` because that's variable.

Reply:
Will update the doc to reflect <path>/<pid>/data as you had indicated to make it clearer.